.github/workflows/packages-e2e-tests.yaml

```diff
@@ -37,19 +41,71 @@ jobs:
         id: tag
         run: echo "tag=$(make version)" >> $GITHUB_OUTPUT
 
-      # Logging into Docker Hub makes the job less flaky, but it relies on repo secrets so
-      # does not work for external contributors. Let's compromise and do this for internal
-      # PRs at least (for now). We can always improve the situation later.
-      - name: Login to Docker Hub
-        if: ${{ github.event.pull_request.head.repo.full_name == 'cilium/tetragon' }}
-        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
-        with:
-          username: ${{ secrets.DOCKER_HUB_USERNAME_CI }}
-          password: ${{ secrets.DOCKER_HUB_PASSWORD_CI }}
-
       - name: Generate Tetragon Tarball
         id: tetragon-tarball
-        run: make tarball
+        run: |
+          make tarball
+          mkdir ${{ matrix.upload_path }}
+          mv ./build/${{ matrix.arch }}/linux-tarball/tetragon-${{ steps.tag.outputs.tag }}-${{ matrix.arch }}.tar.gz ./${{ matrix.upload_path }}
+        if: ${{ matrix.cross_compile == 'no' }}
+
+      - name: Generate Cross Compiled Tetragon EE Tarball
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y qemu qemu-user-static binfmt-support
+          sudo update-binfmts --display
+          make TARGET_ARCH=${{ matrix.arch }} tarball
+          mkdir ${{ matrix.upload_path }}
+          mv ./build/${{ matrix.arch }}/linux-tarball/tetragon-${{ steps.tag.outputs.tag }}-${{ matrix.arch }}.tar.gz ./${{ matrix.upload_path }}
+        if: ${{ matrix.cross_compile == 'yes' }}
+
+      # Cache tarball releases for later
+      - name: Save tetragon-${{ steps.tag.outputs.tag }}-${{ matrix.arch }}.tar.gz Tarball
+        uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
+        with:
+          name: tetragon-${{ steps.tag.outputs.tag }}-${{ matrix.arch }}
+          path: ${{ matrix.upload_path }}
+          retention-days: 1
```

Comment:
if you want you could increase the retention-days so that we have the thing available in case of failure and run a delete artifact on success so that we don't fill the storage for nothing (like we do in vmtests).

Reply:
I didn't get the delete artifact (no idea how it work ;-) , but retention-days 1 is minimum last time I checked, and I don't think we want to keep the tarball, the build is logged in the previous job, so we are covered

Reply:
Okay!

Delete works like this:
https://github.com/cilium/tetragon/blob/1a84735f6b21a9c5c58c82c18baf8311664572f8/.github/workflows/vmtests.yml#L159-L163